Dockerfile

```diff
@@ -41,3 +38,18 @@ RUN sudo apt-get -y clean && sudo rm -rf /var/lib/apt/lists/*
 
 ENV JAVA_IBM8_HOME=/opt/ibm/java/jre \
     IBM_JAVA_OPTIONS="-XX:+UseContainerSupport"
+
+#Set some odd looking variables, since their default values are wrong and it is unclear how they are used
+ENV JAVA_DEBIAN_VERSION=unused
+ENV JAVA_VERSION=unused
+
+# Make java8 a default jvm
+RUN sudo update-java-alternatives -s java-1.8.0-openjdk-amd64
```

Comment:
My simple understanding of docker... I think it results in a more optimal container if we can bundle `RUN` commands together.

Reply:
I think this is slightly easier to read.
I've never heard about 'optimal container' idea.
I've heard about the fact that we should put things that change less often towards to the top to reduce number of layers that need rebuilding. But I feel this is less applicable to our case simple because we usually want things rebuilt anyway - and we usually do rebuilds very infrequently.